RockWeb/Blocks/Communication/CommunicationEntry.ascx.cs

```diff
@@ -1257,6 +1270,26 @@ private Rock.Model.Communication UpdateCommunication(RockContext rockContext)
             return communication;
         }
 
+        private void ValidateFutureDelayDateTime()
+        {
+            DateTime? futureSendDate = dtpFutureSend.SelectedDateTime;
+            TimeSpan? futureTime = dtpFutureSend.SelectedTime;
+
+            if ( futureTime.HasValue && !futureSendDate.HasValue )
+            {
```

Comment:
Can you have this check that both the date and time have been entered i.e. add `|| (futureSendDate.hasValue && !futureTime.HasValue)`